src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav_bar {
+  margin-right: 50px;
+}
+
+.logo {
+  text-decoration: none;
+  line-height: 0;
+  margin-left: 50px;
+}
+
+.item_box {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  align-items: center;
+  align-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.item_box :nth-child(1) {
+  margin: 0;
+}
+
+.item {
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+}
+
+.item > .is-active:first-of-type {
+  color: rgb(0, 172, 219);
```

Comment:
Use `hex` and this color should be stored in a variable as you use it so many times
"CSS Variable is used for a blue color"